tools/packaging/kata-deploy/Dockerfile

```diff
@@ -20,7 +20,9 @@ RUN \
 	mkdir -p ${DESTINATION} && \
 	tar xvf ${WORKDIR}/${KATA_ARTIFACTS} -C ${DESTINATION} && \
 	rm -f ${WORKDIR}/${KATA_ARTIFACTS} && \
-	apk del curl
+	apk del curl && \
+	apk --no-cache add jq py3-pip && \
+	pip install yq
```

Comment:
And this is a bit confusing - `tomlq` relies on `jq`, and is installed as part of the pip version of `yq` could we add this to the commit message at least so we/I don't get confused about this later?

Reply:
Sure, I can do that!

Reply:
Done!